src/panels/lovelace/editor/config-elements/hui-sensor-card-editor.ts

```diff
@@ -152,28 +154,24 @@ export class HuiSensorCardEditor extends LitElement
             )} (${this.hass.localize(
               "ui.panel.lovelace.editor.card.config.optional"
             )})"
-            .value="${this._unit}"
-            .configValue="${"unit"}"
-            @value-changed="${this._valueChanged}"
-          ></paper-input>
-          <paper-input
-            .label="${this.hass.localize(
-              "ui.panel.lovelace.editor.card.sensor.graph_detail"
-            )} (${this.hass.localize(
-              "ui.panel.lovelace.editor.card.config.optional"
-            )})"
-            type="number"
-            .value="${this._detail}"
-            .configValue="${"detail"}"
-            @value-changed="${this._valueChanged}"
+            .value=${this._unit}
+            .configValue=${"unit"}
+            @value-changed=${this._valueChanged}
           ></paper-input>
+          <ha-formfield label="Show more detail">
+            <ha-switch
+              .checked=${this._detail === 2}
+              .configValue=${"detail"}
+              @change=${this._change}
+            ></ha-switch>
```

Comment:
Do we want this to be a switch or a slider?

Reply:
Slider? Its only 2 values

Reply:
Detail or no detail

Reply:
oh... why it is a number then? 😝

Reply:
no idea. Mentioned that above. 0 idea why this was made a number. I converted it to Typescript. Can't remember if I created this or not.

Reply:
Oh, should I read the description of the PR? 🙈